### PR TITLE
FMD-218: paketo build

### PR DESCRIPTION
**Pull request by gidsg**
### Change description
Sister PR to: https://github.com/communitiesuk/funding-service-design-post-award-submit/pull/81

Switches image build to paketo buildpack and makes associated changes to static assets build process.

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Have deployed and tested on dev environment: https://github.com/communitiesuk/funding-service-design-post-award-data-frontend/actions/runs/8017127435/job/21900428844

When run locally CSS and JS should be hot-reloaded. But on deployed environments it will only happen at build time to satisfy contraints of Paketo.

### CODE REVIEW

**Comment by gidsg** [comment]
pre-commit.ci autofix

**Review by MarcUsher — APPROVED** [review]
LGTM but might be worth a check by someone who's had more experience with the front end?

File: build.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import os
+import shutil
+import urllib.request
+import zipfile
+
+import static_assets
+
+
+def build_govuk_assets(static_dist_root="app/static/dist"):
+    GOVUK_FRONTEND_VERSION = "4.6.0"
```
Comment (Joel-Pearce) [inline build.py:11]: The most up-to-date version is actually ```4.7.0```. This is the version in the Submit repo. Do we want to standardise the two?

**Review by Joel-Pearce — CHANGES_REQUESTED** [review]

File: build.py (hunk 1 above)
Comment (gidsg) [reply build.py:11]: The latest version is actually 5.2.0! https://github.com/alphagov/govuk-frontend/releases

We could standardise it, just wasn't strictly required to upgrade for this change, so maybe could be picked up separately?

Comment (gidsg) [reply build.py:11]: @Joel-Pearce for now I kept this as per the previous build script: https://github.com/communitiesuk/funding-service-design-post-award-data-frontend/blob/8a22d458c9015fe973d0ccc938f75d05e4b24d03/build.sh#L7 maybe we could go up to the latest `4.x` release which shouldn't introduce breaking changes, but it's not strictly required for this change so didn't want to cause a change in behaviour inadvertently.

**Pull request merged**